controllers/grid/users/reviewer/form/AdvancedSearchReviewerForm.php

```diff
@@ -211,6 +211,21 @@ public function fetch($request, $template = null, $display = false)
             ]
         ]);
 
+        // Get the submission's authors
+        $publication = $this->getSubmission()->getCurrentPublication();
+
+        $authors = [];
+        foreach($publication->getData('authors') as $author) {
+            $affiliations = [];
+            foreach($author->getAffiliation(null) as $affiliationName) {
```

Comment:
Use `$author->getData('affiliation')` instead of `$author->getAffiliation(null)`